awscli/argprocess.py

```diff
@@ -245,13 +245,36 @@ def _is_complex_shape(model):
     return True
 
 
+def _uses_old_list_case(service_name, operation_name, argument_name):
```

Comment:
Any reason this needs to be at the module level?  Seems like this is specific to the shorthand parsing class, so it makes sense in there.